lib/note-list/index.tsx

```diff
@@ -320,6 +339,7 @@ export class NoteList extends Component<Props> {
                     rowHeight={heightCache.rowHeight}
                     rowRenderer={renderNoteRow}
                     scrollToIndex={selectedIndex}
+                    tabIndex={null}
```

Comment:
what's the difference between `tabIndex={-1}` and `tabIndex={null}`?

Reply:
The difference being that `tabIndex={null}` unsets a `tabIndex="0"` [set by default from `react-virtualized`](https://github.com/bvaughn/react-virtualized/issues/275#issuecomment-226077606) and `tabIndex={-1}` disables the element from being focused by keyboard tabbing while still allowing programmatic focus.

Given that the contents of our virtualized list are focusable (buttons to pin or edit a note), IMO it doesn't make sense for the entire list to be a focusable element. We also, to my knowledge, have no need to programmatically focus the list, so `null` makes more sense than `-1`.

Reply:
Good question. I had not explicitly tested scrolling the list with the keyboard until just now. It would appear the scrolling the list is still possible and matches the experience as when `tabindex="0"` is set. 👍🏻